### DISCUSSION TITLE
Impression of an absolute beginner

### DISCUSSION THREAD

**Discussion by BsNoSi** [Zettlr/Zettlr#2604] [General]
I installed Zettlr today and was very impressed from version 1.8.9. 
After some tests, I was curious about the 2.0beta5. I returned after a few attempts to version 1.8.9

**For me as an absolute beginner, version 2 Beta is a back step.** 

I have to admit that the lack of current documentation for version 2 may be a reason for my failure. But If there is no backward compatibility, I would appreciate a sample, explaining what has to be done to achieve the previous results.  Have I overlooked them?

**My impressions**

- It is much harder to achieve intended (export) results, exporting a project to PDF with own template failed (no file generated with pdf export).  This was a breeze with version 1.8.9.
- Configuration via assets may offer more options, but it makes it even harder to get results without forcing yourself to make you an  expert of each additionally used tool. Because of this requirement, I refrained at this point going deeper into version 2.
- Beta looks in comparison to 1.8.9 on Windows untidy, unclear and is - for me - confusing. 
I strongly hope, that V2 will return to the much cleaner (beginner) interface of version 1, its ease of use, to achieve adjusted results again without the need of going deeply into detailed technical aspects.

> I would like to return to beta if anybody can tell me what I have to do to achieve a pdf project output with an individual template. This is a personal requirement for the use of Zettlr.

If it makes sense for myself using Zettlr, I will gladly support the community with documentation. Especially "how tos"  for absolute beginners. Today I became an impression of the feeling, finding a great tool, but being unable to get profit of it, because there is no explanation (on my level for V2). I would like to remedy this.

**Comment by nathanlesage** [comment]
Hey! First, welcome to the community and thank you for being so brave and immediately trying out one of the betas!

Then you are absolutely right: The documentation is almost at no point prepared for 2.0 and a lot of things have changed, but since we only have one single documentation, I still had to somehow maintain it compatible with 1.8.9 so we're right now in this weird twilight state where the documentation is not really satisfying for anyone. However, I intend to fix that in the coming weeks.

Then, there were still a few minor bugs in the beta.5 which I addressed since. I'm very sure that with the slowly progressing documentation it will become easier to adapt all the settings, because even though the defaults in the assets manager look daunting at first, they give you much more freedom. Yes, we (I) do need to document them better, but then it should be very simple to adapt them. However, right now they're almost not documented, because even the Pandoc documentation isn't really great at that, so I fully understand your frustration in this regard.

With regard to the template, it might be that I simply forgot to add that option to the preferences dialog. Argh, yes I did. I will make sure to (re)add that option to the projects settings before I release 2.0 stable.

Until then, please feel free to also [join our community discord](https://discord.gg/PcfS3DM9Xj); there are many people with lots of experience and me, who will all gladly help you with any questions.

**Comment by BsNoSi** [comment]
You got me. This morning I returned to Beta. Discord is installed. Hopefully you will not regret this invitation one day ;)

One question: Where shall I place what?
- Issues → Github
- Discussions → Github or Discord? What makes the difference to prefere one of them for it?

I have to admit that I am a quite *old school user* – I preferer classic forum styles in a browser and e-mail as "exchange format" for information. Real-time communication via chat is difficult to organize and filter when it comes to development issues. Discord is new to me and it is therefore unclear what possibilities  or advantages it might offer.

**Reply by nathanlesage** [reply]
> Hopefully you will not regret this invitation one day ;)

Hehehe, don't worry!

Choose whatever you like; part of the reason there are so many different platforms is because I wannat make sure many are able to participate in the community discussions! In general, I found that Discord and reddit are much easier for quick'n'dirty questions (that is, specifically where the docs are still not correct as of now), whereas GitHub and the Discussions section here is ideal for somewhat longer discussions! :)
